src/components/atoms/Icon/Icons/ellipse.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="16" height="26" viewBox="0 0 16 26" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path d="M16 8C16 12.4183 8 25.5 8 25.5C8 25.5 0 12.4183 0 8C0 3.58172 3.58172 0 8 0C12.4183 0 16 3.58172 16 8Z" fill="#00BEBE"/>
```

Comment:
Please make the fill `currentColor` and then pass the color into the Icon component. This way we can control the color of the icon based on the selected theme.

Reply:
I fixed! Thank you!
[fix: use currentColor in svg](https://github.com/reearth/reearth-web/pull/256/commits/a7de59828bd1c063ffdbd2d857f0e722059b3964)